CorsixTH/com.corsixth.corsixth.metainfo.xml

```diff
@@ -26,6 +26,15 @@
   </provides>
   <launchable type="desktop-id">com.corsixth.corsixth.desktop</launchable>
   <releases>
+    <release version="0.65.1" date="2021-07-13" type="stable">
+      <description>
+        <p>This is a hotfix release of 0.65 and replaces the June 2021 release</p>
+        <ul>
+          <li>Fixes a bug where the game would crash if you got the All Cured trophy at the
```

Comment:
Important context is that this is only when loading a save from 0.64

Reply:
Its not 0.64 specific. But point noted I need to say for existing saves